src/common/Defs.h

```diff
@@ -46,7 +46,13 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define GAMENAME_STRING     "unv"
 #define GAMENAME_FOR_MASTER PRODUCT_NAME_UPPER
 
-#define MASTER_SERVER_NAME  "master.unvanquished.net"
+#define MASTER1_SERVER_NAME "master.unvanquished.net"
+#define MASTER2_SERVER_NAME "master2.unvanquished.net"
+#define MASTER3_SERVER_NAME ""
+#define MASTER4_SERVER_NAME ""
+#define MASTER5_SERVER_NAME ""
+#define MOTD_MASTER_SERVER_NAME     MASTER1_SERVER_NAME
```

Comment:
Doesn't this mean that if master 1 goes down, these two won't work. I'd rather move back to a cvar scheme...

Reply:
This was already a case and worst than that, it was using the hardcoded first master server even if the first master server cvar was updated. There was a similar behavior for the gamestat sending (sending to the first hardcoded master server only). See my new commits attempting to fix that.